docs/loading/SparkLoad.md

```diff
@@ -6,8 +6,7 @@ Spark load is an **asynchronous** import method that requires users to create Sp
 
 > **NOTICE**
 >
-> - You can load data into StarRocks tables only as a user who has the INSERT privilege on those StarRocks tables. If you do not have the INSERT privilege, follow the instructions provided in [GRANT](../sql-reference/sql-statements/account-management/GRANT.md) to grant the INSERT privilege to the user that you use to connect to your StarRocks cluster.
-> - When Spark Load is used to load data into a StarRocks table, the bucketing column of the StarRocks table cannot be of DATE, DATETIME, or DECIMAL type.
+> Only users with the INSERT privilege on a StarRocks table can load data into this table. You can follow the instructions provided in [GRANT](../sql-reference/sql-statements/account-management/GRANT.md) to grant the required privilege.
```

Comment:
suggestion: 
You can follow the instructions provided in [GRANT](../sql-reference/sql-statements/account-management/GRANT.md) to grant the required privilege. -> You can use the [GRANT](../sql-reference/sql-statements/account-management/GRANT.md) statement to grant the INSERT privilege to a user or role.